src/data/thermal-conductivity-constants.ts

```diff
@@ -16,8 +16,374 @@ export const DIPPRThermalConductivityDictionary: Record<
     thermalConductivityAtMinimumTemperature: 0.2524,
     thermalConductivityAtMaximumTemperature: 0.1727,
   },
+  Acetamide: {
+    name: 'Acetamide',
+    formula: 'C2H5NO',
+    CAS: '60-35-5',
+    molecularWeight: 59.0672,
+    C1: 0.39363,
+    C2: -0.00037053,
+    minimumTemperature: 353.33,
+    maximumTemperature: 494.3,
+    thermalConductivityAtMinimumTemperature: 0.2627,
+    thermalConductivityAtMaximumTemperature: 0.2105,
+  },
```

Comment:
I think we should push a new line after a compound data is mapped will be easier to read

Reply:
Sir @maneetgoyal  should we do this i have done this in previous PR's?

Reply:
Yeah, sounds good